SAGESharp/XFile.cs

```diff
@@ -51,19 +51,19 @@ public void Write(BinaryWriter writer)
 
             foreach (XTemplate template in Templates)
             {
-                Console.WriteLine("    Writing template '" + template.Name + "'...");
+                //Console.WriteLine("    Writing template '" + template.Name + "'...");
```

Comment:
Can we either remove these debugging statements or guard them behind some compiler symbol?